Heat_method_3/include/CGAL/Heat_method_3/internal/Intrinsic_Delaunay_triangulation_3.h

```diff
@@ -37,6 +37,7 @@
 #include <set>
 #include <stack>
 #include <cmath>
+#include <list>
```

Comment:
That is used by:
```
  typedef typename std::stack<edge_descriptor, std::list<edge_descriptor> > edge_stack;
```
Should not that stack use `std::deque` instead of a list?!

Reply:
The question is for @janetournois

Reply:
`std::deque` is the default, so I wonder why we would have chosen something else in the first place...

Reply:
It's been there for a while, and using `git blame` does not help much